src/components/split-button/split-button.style.js

```diff
@@ -0,0 +1,135 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import StyledButton from '../button/button.style';
+import BaseTheme from '../../style/themes/base';
+import colors from './split-button-colors.style';
+import OptionsHelper from '../../utils/helpers/options-helper';
+
+const StyledSplitButtonContainer = styled.div`
+  display: inline-block;
+  position: relative;
+  &:focus {
+    z-index: 20;
+  }
+`;
+// Need to size the main button
+
+export const StyledMainButton = styled(StyledButton)`
```

Comment:
Still to do?

Reply:
Yep, something is defaulting the style of the main button atm, so may just have to define some styles to override the default